common/changes/@microsoft/rush/chao-update-pnpm-config_2023-08-30-00-23.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "feat: add resolutionMode in pnpm-config",
```

Comment:
```suggestion
      "comment": "(IMPORTANT) Add a new setting `resolutionMode` in pnpm-config.json; be aware that Rush now overrides the default behavior if you are using PNPM 8.0.0 through 8.6.12 (GitHub #4283)",
```

Reply:
FYI there's some tips for writing `rush change` messages in this doc: [Authoring change logs](https://rushjs.io/pages/best_practices/change_logs/)